package-builder/flox-build.mk

```diff
@@ -237,15 +242,17 @@ define BUILD_local_template =
 	  $(FLOX_INTERPRETER)/activate --env $(FLOX_ENV) --mode dev --turbo -- \
 	    $(_env) -i out=$(_out) $(foreach i,$(ALLOW_OUTER_ENV_VARS),$(i)="$$$$$(i)") \
 	      $(_build_wrapper_env)/activate --env $(_build_wrapper_env) --mode dev --turbo -- \
-	        $(_bash) -e $($(_pvarname)_buildScript)
-	$(_V_) set -o pipefail && $(_nix) build -L --file $(_libexec_dir)/build-manifest.nix \
+	        $(_t3) --forcecolor --relative $($(_pvarname)_logfile) -- $(_bash) -e $($(_pvarname)_buildScript)
+	$(_V_) $(_nix) build -L --file $(_libexec_dir)/build-manifest.nix \
```

Comment:
does forcecolor work on NO_COLOR or incompatible terminals, i.e.e is there any detection built into t3 other than "is tty or not"?

Reply:
We can certainly tweak the `t3` flags to suit our purposes, but like any good filter `t3` disables color output when not talking to a terminal, and `--forcecolor` disables this disabling. I had to add that because even though the flox CLI is relaying all output to STDOUT/STDERR for the build command, it is in fact not a tty, so that was the only way to get color output. If we can instead arrange for the flox CLI to pass STDOUT/STDERR unaltered to the `make` invocation then we can drop that flag.

FWIW the lines written to the log file will always contain color unless invoked with the `--plain` flag, in the knowledge that it's always possible to remove ANSI color sequences if necessary, but not possible to add them back in after the fact.

Reply:
Added support to honor `NO_COLOR` in commit 87a9cb9506ba464ef81d7e8e6b164f42767504f8. Resolving conversation - thanks @ysndr for the suggestion!